### PR TITLE
minor modification

### PR DESCRIPTION
**Pull request by chrishobcroft**
I couldn't get this to work, but there are some dependencies I added, and some other stuff

### CODE REVIEW

File: Workflow.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
 # Workflow
+
+## Pre-requisites
+
+- Node (latest LTS version), with the /bin folder on your $PATH
+- aragonCLI
```
[the remark this answers is not in the document]
Comment (xseignard) [reply Workflow.md]: I don't use `npx` but I think both approaches are OK.
I've mixed feelings about `aragon ipfs` being a pre-requisite, it's a cli invocation, not a pre-requisite.
What's the point of saying you have the `/bin` folder on your `$PATH` ?

I get the need of having a pre-requisite section, but this PR is not fulfilling it.

[the remark this answers is not in the document]
Comment (deamme) [reply Workflow.md]: Yeah. I think it will be better to delete the `Pre-requisites` section and this PR should be good

**Comment by chrishobcroft** [comment]
As you like. But it helps to be clear about what the pre-requisites are for running this process. It seems that at least node is required before you can start. Anything else?

**Comment by deamme** [comment]
@chrishobcroft It's true and it also depends on the target group. For me it's pretty obvious that it's a Node.js project and developers trying to run it would already have it. @xseignard What are your thoughts?

**Pull request closed without merging**